barretenberg/cpp/src/barretenberg/vm2/tracegen_helper.cpp

```diff
@@ -238,7 +238,11 @@ TraceContainer AvmTraceGenHelper::generate_trace(EventsContainer&& events)
             std::make_unique<LookupIntoIndexedByClk<lookup_bytecode_to_read_unary_lookup_settings>>(),
             std::make_unique<LookupIntoIndexedByClk<lookup_bytecode_bytes_are_bytes_lookup_settings>>(),
             std::make_unique<LookupIntoIndexedByClk<lookup_bytecode_remaining_abs_diff_u16_lookup_settings>>(),
-            std::make_unique<LookupIntoIndexedByClk<lookup_sha256_round_constant_lookup_settings>>());
+            std::make_unique<LookupIntoIndexedByClk<lookup_sha256_round_constant_lookup_settings>>(),
+            /* Bytecode Hashing */
```

Comment:
oh yes, I was craving for these separations for a long time.
However, I'd prefer if you use `//` since it's what we use the most :P